src/renderers/shared/fiber/__tests__/ReactChildFiber-test.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+var ReactDOMFeatureFlags = require('ReactDOMFeatureFlags');
+
+var React;
+var ReactTestUtils;
+var ReactFeatureFlags;
+
+describe('ReactChildFiber', () => {
```

Comment:
For our new test suites, we're trying to give them names that are less coupled to our module structure and implementation details. Maybe `ReactChildren` instead? Not a huge deal